community/examples/hpc-build-slurm-image.yaml

```diff
@@ -0,0 +1,116 @@
+# Copyright 2024 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+blueprint_name: hpc-build-slurm-image
+
+vars:
+  project_id: ns-playground-2023-01-19  ## Set GCP Project ID Here ##
+  deployment_name: build-slurm-1
+  region: us-central1
+  zone: us-central1-a
+
+  image_build_machine_type: n2d-standard-32
+  build_from_image_family: hpc-rocky-linux-8
+  build_from_image_project: cloud-hpc-image-public
+  built_image_family: my-custom-slurm
+  built_instance_image:
+    family: $(vars.built_image_family)
+    project: $(vars.project_id)
+  instance_image_custom: true
+
+deployment_groups:
+- group: setup
+  modules:
+  - id: network
+    source: modules/network/vpc
+
+  - id: slurm-build-script
+    source: modules/scripts/startup-script
+    settings:
+      runners:
+      - type: shell
+        destination: prep-for-slurm-build.sh
+        content: |
+          #!/bin/bash
+          set -e -o pipefail
+          # Slurm build on Rocky8 will upgrade to python38 as part of build
+          # This is not compatible with ansible-local runner
+          dnf install -y python38
+          alternatives --set python3 /usr/bin/python3.8
+          python3 -m pip install pip --upgrade
+          python3 -m pip install ansible
```

Comment:
Is version of Ansible critical? Here you are installing a very recent copy of Ansible (good) but the rest of our Ansible solutions are tested with 2.11.

If the user were to add Ansible runners for the actual cluster, it would run on the copy of Ansible you are installing here. It might be wise to align with rest of Toolkit even if that mans an older version.

What do you think?